examples/flight/.gitignore

```diff
@@ -1 +1,40 @@
-node_modules/
+node_modules/.bin/*
+
+node_modules/depot/*
```

Comment:
The start at the end isn't really needed. Ignoring the folder itself will work too.